english/coffee/coffee.md

```diff
@@ -9,3 +9,5 @@ After having your coffee do you want to:
 [Write a caffeine-fueled full-length novel?](novel/full-length-novel.md)
 
 [Go skinny dipping?](nude-run/nude-run.md)
+
+[Light a Fire?](light-fire/fire.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.